data/hai/hai reveal 5 philosophers.txt

```diff
@@ -848,19 +945,62 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
-			`	Before you all leave for the invasion operation, General Choot'k finds you making final preparations for launch. "You have handled yourself well throughout this. I want to see you prove yourself in battle before I open the entrance to <planet>. When we arrive at <planet>, we'll first fight the defense fleet waiting for us. I suspect we can handle it, but you are of course welcome to assist. The main priority for you should be to make sure nobody pursues Elder Teeneep or Captain Wallace. However, these scum will likely keep a couple of ships in reserve. The scans you took suggest some are not totally fighting fit, or perhaps will be kept to use their weapons inside the spaceport as a backup defense. To counter this, you'll need to draw them out by demanding tribute from the city. They'll keep sending defense fleets until they resign to their fate and pay you daily tribute. This is where we would like you to prove yourself. We won't help you fight the tribute defense fleets, but we'll fight the pirates waiting for our arrival and any other pirates that show up."`
-			`	He seems to admire your ship for a moment. "Tell the other humans not to help you either. If you succeed today you shall be Hai-friend, wherever you go."`
+			`	"Of course, nobody tells me anything in a timely manner..."`
+			`	"Anyway, I've organized the Syndicate's 'hostile urban insertion security operatives' to be loaded onto the Auxiliary. They look like intelligence operatives, if you ask me, so that'll be a conversation for me to follow up on later, but for now they're specially trained for attacking pirate strongholds and equipped with state of the art equipment that... is on a need-to-know basis. As long as we can protect the Auxiliary we'll be fine. The intel you brought back doesn't suggest particularly strong resistance. Wallace, good luck."`
+			choice
+				`	"It's good to hear you have the ground combat situation under control."`
+					goto control
+				`	"That equipment sounds ominous."`
+			`	Kopkow gives you a disturbing smile that has an 'I love my work' appearance. He departs as everyone else assembles.`
+				goto leave
+			label control
+			`	"It's more of a collaboration, really, but like I said: need-to-know basis." He nods to you and departs as everyone else assembles.`
+			label leave
+			`	After a large meeting about logistics, and an unsurmountable buffet, everyone disperses. General Choot'k finds you to discuss critical tactical matters.`
+			`	"There's a problem with the plan," he says, "which we found after more analysis of the scans. The bay where we need to land already has ships inside, and if we try to fly in there, they'll blow us to pieces. You have lure them out and kill them all."`
+			choice
+				`	"How do I lure them out?"`
+					goto how
+				`	"Can't we blow them up on the way down?"`
+			`	"Have you ever tried to land on a pile of blown-up space ships in the wreck of a spaceport a half kilometer underground? It ain't easy.`
```

Comment:
I can't help feeling Choot'k feels out of character for saying "ain't" without having it explained to me.

Reply:
yeah he shouldnt be using contractions I think, as a Hai and Unfettered on top of that

Reply:
Choot'k now responds to your surprise.